recipe/meta.yaml

```diff
@@ -32,10 +29,14 @@ test:
   imports:
     - zodbpickle
     - zodbpickle.tests
+  commands:
+    - pip check
+  requires: 
+    - pip
 
 about:
   home: http://pypi.python.org/pypi/zodbpickle
-  license: PSF 2 and ZPL 2.1
```

Comment:
Why the change in license?

Reply:
Previously the bot complained: License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression. Is there a way to have both licenses?

Reply:
Yes. See the SPDX syntax on their site and in out documentation. https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions